### DISCUSSION TITLE
How do percentiles help capture spatially lagged characters

### DISCUSSION THREAD

**Discussion by adhamenaya** [pysal/momepy#376] [Q&A] [answered]
Hi - I was doing the example of [Simplifed detection of urban forms ](http://docs.momepy.org/en/stable/examples/clustering.html). I do appreciate sharing this useful example. Still, I don't get the idea of using percentiles to describe the spatially lagged values. Could anyone please share more information or put a reference that may help understand it? Thank you.

**Comment by martinfleis** [comment, marked as answer]
Hi, the idea behind that is relatively simple. Traditional spatial lag is a mean of values defined as `neighbors`. However, in reality, urban environment with spatial weights matrices spanning more than first order, have a tendency to show non-normal distributions. Often power-law like (e.g. networks), bi-modal (e.g. street widths) or other. We are trying to capture this property by sampling from each distribution using percentiles. Then two places may have the same mean but different percentiles and that is what we want to capture.

**Reply by adhamenaya** [reply]
Thanks a lot, dear @martinfleis. It's very helpful.
